modules/videoio/test/test_container_avi.cpp

```diff
@@ -0,0 +1,87 @@
+// This file is part of OpenCV project.
+// It is subject to the license terms in the LICENSE file found in the top-level directory
+// of this distribution and at http://opencv.org/license.html.
+
+#include "test_precomp.hpp"
+#include "opencv2/videoio/container_avi.private.hpp"
+#include <cstdio>
+
+using namespace cv;
+
+namespace opencv_test
+{
+
+TEST(videoio_avi, good_MJPG) {
+    String filename = cvtest::TS::ptr()->get_data_path() + "video/big_buck_bunny.mjpg.avi";
```

Comment:
You should add `BunnyParameters::getFilename(".mjpg.avi");` here for consistency.